data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -274,7 +294,8 @@
       "NO_REPAIR",
       "NO_SALVAGE",
       "NO_UNLOAD",
-      "NO_AMMO"
+      "NO_AMMO",
+      "MAGIC_FOCUS"
```

Comment:
i'm not sure i want to make the woodbow even more powerful. it's already super easy to cast (low tier spell) and has incredible damage output.

Reply:
Oh, I wouldn't worry about it. now that we have the spell chaining I'm going to rework that and update it with the rest of the spells I'm building, put in a quiver w/arrows and all that, there's no other mechanic for me to use to make it more balanced.

Reply:
I agreed that doing a mana per shot would be a great way of handling it, but that's not possible rn, and I'm fine with limited shots. There was just no other way to make it work.

Reply:
then it should probably get the change when that is updated, not be even more overpowered in the meantime.